assets/src/edit-story/index.js

```diff
@@ -31,19 +32,25 @@ import './style.css'; // This way the general editor styles are loaded before al
  *
  * @param {string} id       ID of the root element to render the screen in.
  * @param {Object} config   Story editor settings.
+ * @param flags
  */
-const initialize = (id, config) => {
+const initialize = (id, config, flags) => {
   const appElement = document.getElementById(id);
 
   // see http://reactcommunity.org/react-modal/accessibility/
   Modal.setAppElement(appElement);
 
-  render(<App config={config} />, appElement);
+  render(
+    <FlagsProvider features={flags}>
+      <App config={config} />
+    </FlagsProvider>,
+    appElement
+  );
 };
 
 const initializeWithConfig = () => {
-  const { id, config } = window.webStoriesEditorSettings;
-  initialize(id, config);
+  const { id, config, flags } = window.webStoriesEditorSettings;
```

Comment:
Should we have a simpler model to turn the flags for development? E.g. pass them as an arg --flags and merge them into the final list?

Reply:
Why is that easier than modifying the Story_Post_Type.php file? I think that would be necessary if/when this application runs outside of Wordpress.

Reply:
Maybe not much easier, but felt a bit simpler to me w/o requiring me to modify a source file and hoping not to accidentally commit it. The php file, though, is good for launches.